linera-execution/src/evm/revm.rs

```diff
@@ -583,12 +593,13 @@ enum Choice {
 #[derive(Debug)]
 struct ExecutionResultSuccess {
     reason: SuccessReason,
+    gas_final: u64,
```

Comment:
We should consider adding a dedicated type for this and make sure all operations are checked.

Reply:
The thing is that REVM does not have a dedicated type for the gas. So, I do not see what can be achieved with that.

We could of course ask REVM to do switch to nested type, but what would they get from that? Only slowdown and not any improvement.

And REVM is doing subtractions that are unchecked. So, in think that we cannot achieve much that way.